cmd/utils/flags.go

```diff
@@ -96,7 +96,7 @@ func NewApp(gitCommit, usage string) *cli.App {
 	//app.Authors = nil
 	app.Email = ""
 	app.Version = params.Version
-	if gitCommit != "" {
+	if gitCommit != "" && len(gitCommit) >= 8 {
```

Comment:
You can drop the `gitCommit != ""` part, `len(gitCommit) >= 8` already validates that it's not `""`.